controller/experiment.controller.js

```diff
@@ -19,27 +20,27 @@ const getExperimentById = async (req, res) => {
     res.status(200).json(result);
 };
 
-const getExperimentsByAcaccountId = async (req, res) => {
-    if (!req.params.acaccountId) throw new PropertyNotFound("acaccountId");
-    const acaccountId = req.params.acaccountId;
-    const result = await ExperimentRepository.findByAttribute("acaccountId", acaccountId);
-    if (!result) throw new ServerUnableError("getExperimentsByAcaccountId")
+const getExperimentsByAccountId = async (req, res) => {
+    if (!req.params.accountId) throw new PropertyNotFound("accountId");
+    const accountId = req.params.accountId;
+    const result = await ExperimentRepository.findByAttribute("accountId", accountId);
+    if (!result) throw new ServerUnableError("getExperimentsByAccountId")
     res.status(200).json(result);
 }
 
 const getExperimentsAB = async (req, res) => {
-    if (!req.params.acaccountId) throw new PropertyNotFound("acaccountId");
-    const acaccountId = req.params.acaccountId;
```

Comment:
You misspelled accountId everywhere =]

Reply:
I noticed these typos everywhere in the code and tried to fix them. maybe missed some. I will do a check again